<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DuplicateItem</class>
 <widget class="QDialog" name="DuplicateItem">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>180</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QWidget {
	background-color: rgb(191,193,194);
	border-color: black;
}
*/
/*	Push button*/
QPushButton {
	min-height: 20px;
	border: 1px solid gray;
	border-radius: 3px;
	background-color: lightgray;
	selection-background-color: gray;
}

QPushButton:hover {
	background-color: gray;
	border-radius: 3px;
}


/*Tool button*/
QToolButton {
	max-width: 20px;
	max-height: 20px;
	border: 0px solid gray;
}

/*Line edit*/
QLineEdit {
    border: 1px solid gray;
	border-radius: 3px;
    background-color: lightgray;
    selection-background-color: darkgray;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="m_pFrameTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>290</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(105,105,105)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="m_pLabelTitle">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Duplicate Item Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_pToolButtonClose">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../Resources/Icons/close-icon.png</normaloff>../../../../Resources/Icons/close-icon.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_pVerticalWidgets">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(105,105,105)
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(191,193,194);
border-radius: 3px;

</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>7</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>Scale:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>Location:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>Rotation:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0" columnstretch="0,0,0,0,0,0,0,0,0">
           <property name="spacing">
            <number>7</number>
           </property>
           <item row="2" column="0">
            <widget class="QLabel" name="label_51">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="8">
            <widget class="QLineEdit" name="m_pLineEditRotationZ">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="m_pLineEditRotationX">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="m_pLineEditLocationX">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="5">
            <widget class="QLineEdit" name="m_pLineEditRotationY">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="m_pLineEditScaleX">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>1.0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="7">
            <widget class="QLabel" name="label_52">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>Z:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_45">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QLineEdit" name="m_pLineEditScaleZ">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>1.0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="8">
            <widget class="QLineEdit" name="m_pLineEditLocationZ">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="7">
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string>Z:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_49">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_53">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLineEdit" name="m_pLineEditScaleY">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>1.0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="label_46">
             <property name="text">
              <string>Z:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLineEdit" name="m_pLineEditLocationY">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="horizontalWidget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(191,193,194);
border-radius: 3px;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>16</number>
         </property>
         <item>
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>Copies:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_pSpinBoxCopies">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>21</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSpinBox {
    border: 1px solid gray;
	border-radius: 3px;
    background-color: lightgray;
    selection-background-color: darkgray;
    padding-right: 15px;
	min-height: 19px;
	max-height: 19px;
}
</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_pFrameButtonBox">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(105,105,105);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_pPushButtonDuplicate">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	min-height: 20px;
	border: 1px solid gray;
	border-radius: 3px;
	background-color: gray;
	selection-background-color: rgb(191,193,194);
}

QPushButton:hover {
	background-color:rgb(105,105,105);
	border-radius: 3px;
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pPushButtonCancel">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	min-height: 20px;
	border: 1px solid gray;
	border-radius: 3px;
	background-color: gray;
	selection-background-color: rgb(191,193,194);
}

QPushButton:hover {
	background-color:rgb(105,105,105);
	border-radius: 3px;
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
